BlueprintUI/Sources/BlueprintView/BlueprintView.swift

```diff
@@ -323,20 +366,28 @@ extension BlueprintView {
             self.viewDescription = node.viewDescription
             self.layoutAttributes = node.layoutAttributes
             self.children = []
+            
             self.view = node.viewDescription.build()
+            self.view.nativeViewNodeBlueprintEnvironment = node.environment
+        }
+        
+        deinit {
+            self.view.nativeViewNodeBlueprintEnvironment = nil
```

Comment:
This is a belt-and-suspenders thing – in _theory_ you should always be returning a new view from your view description; but you don't _have_ to – it could be a static / instance owned elsewhere. So I am clearing the value out when we stop controlling the view just in case.